daemon/algod/api/algod.oas2.json

```diff
@@ -1205,6 +1205,69 @@
         }
       }
     },
+    "/v2/transactions/simulate": {
+      "post": {
+        "tags": [
+          "public",
+          "experimental"
+        ],
+        "consumes": [
+          "application/x-binary"
+        ],
+        "produces": [
+          "application/msgpack"
+        ],
+        "schemes": [
+          "http"
+        ],
+        "summary": "Simulates a raw transaction or transaction group as it would be evaluated on the network. WARNING: This endpoint is experimental and under active development. There are no guarantees in terms of functionality or future support.",
```

Comment:
Can the generator automatically  tack that WARNING onto the summary for any `experimental` API? (I don't feel strongly, until//unless we add another.)

Reply:
We probably wouldn't want the generator to add it automatically since this is its input, but perhaps we could make it generate some asserts to ensure it's present in the summary.

That would involve modifying https://github.com/algorand/oapi-codegen/ though, which is a bit more involved

Reply:
Yes, I didn't mean that it would change this, I meant that we could leave the warning off here, but have the generator append the warning to whatever it creates that uses this text.

Anyway, only worth doing if we have a several experimental things and don't want to remember to edit this when then get promoted.

resolving.